proto/dydxprotocol/govplus/tx.proto

```diff
@@ -1,7 +1,58 @@
 syntax = "proto3";
 package dydxprotocol.govplus;
 
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "amino/amino.proto";
+
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/govplus/types";
 
 // Msg defines the Msg service.
-service Msg {}
+service Msg {
+  // SlashValidator is exposed to allow slashing of a misbehaving validator via
+  // governance.
+  rpc SlashValidator(MsgSlashValidator) returns (MsgSlashValidatorResponse);
+}
+
+// MsgSlashValidator is the Msg/SlashValidator request type.
+message MsgSlashValidator {
+  // The address that controls the module (likely the gov module account).
+  option (cosmos.msg.v1.signer) = "authority";
+  string authority = 1;
+
+  // Address of the validator to slash
+  string validator_address = 2;
```

Comment:
Explicitly name this as `validator_consensus_address` for disambiguation

Reply:
Should also use relevant options for this field. [Example](https://github.com/cosmos/cosmos-sdk/blob/b947b02dc63db0a8011a96e217197bf4559dea92/proto/cosmos/staking/v1beta1/query.proto#L167)?